curvefs/src/mds/fs_info_wrapper.cpp

```diff
@@ -33,23 +35,30 @@ namespace mds {
 
 using ::curvefs::common::S3Info;
 using ::curvefs::common::Volume;
-
-bool FsInfoWrapper::IsMountPointExist(const std::string& mp) const {
-    return std::find(fsInfo_.mountpoints().begin(), fsInfo_.mountpoints().end(),
-                     mp) != fsInfo_.mountpoints().end();
+using google::protobuf::util::MessageDifferencer;
+
+bool FsInfoWrapper::IsMountPointExist(const Mountpoint& mp) const {
+    return std::find_if(fsInfo_.mountpoints().begin(),
```

Comment:
If hostname and path is same, but port is different, it will return exist ?

Reply:
> If hostname and path is same, but port is different, it will return exist ?

Sorry, I changed a lot here, it has been fixed